mnefun/_sss.py

```diff
@@ -1024,3 +1106,20 @@ def annotate_head_pos(raw, head_pos, rotation_limit=45, translation_limit=0.1,
 
     # Annotate on distance from the sensors
     return annot
+
+
+def check_sws():
+    """Check if SSS workstation is configured correctly."""
+    from . import Params
+    p = Params()  # loads config
+    if p.sws_ssh is None:
+        raise RuntimeError('sws_ssh was not defined in mnefun config file %s'
+                           % (_get_config_file(),))
+    output = run_sss_command(None, '-version', None, host=p.sws_ssh,
```

Comment:
If I'm interpreting the code correctly, it is invoking a throw-away call to maxfilter (on the remote server) by just asking for the version.  I didn't initially have the maxfilter path set up, though in the new documentation you nicely request the user to make sure it is.  I haven't yet used maxfilter that way, as I've only used the remote server command for the head position estimation, and other future users may be in the same situation.  Might a call to a more universal remote bash command work, some sort of simple handshaking, or is there a reason to use an MNE-specific command?

Reply:
Maybe I'm misunderstanding your point, but If the test just used a more universal bash command like `$ pwd`, then it wouldn't be checking/confirming that maxfilter is properly installed. So actually running `maxfilter -version` seems preferable to me, anyway.

Reply:
Indeed the idea is to not just make sure that SSH works, but also that the commands we're going to want to run will work (specifically, MaxFilter)